remote/advancedcontainers/docker-options.md

```diff
@@ -16,6 +16,8 @@ You can use Docker with the [Remote - Containers](https://marketplace.visualstud
 * Docker installed on another machine or remote environment.
   * You only need Docker installed on the remote host, rather than Docker installed locally.
 * Other Docker compliant CLIs, installed locally or in a remote environment.
+  * For instance, [Rancher Desktop](https://docs.rancherdesktop.io/) is another way to install Docker, providing container management and Kubernetes on Windows, macOS, and Linux.
```

Comment:
Should we add a link to https://mobyproject.org/ for the Moby reference?

Reply:
Sounds good to me! I added a link.